index.html

```diff
@@ -7,6 +7,7 @@
     <link rel="stylesheet" href="style.css">
     <script defer src="menu.js"></script>
     <script defer src="modal.js"></script>
+    <script defer src="validation.js"></script>
```

Comment:
- [ ] Great job so far 👍 but your email validation does not work as I can still submit the form with an email containing a capital letter. Kindly work on this and make sure that when a user inputs an email containing a capital letter, the form is not submitted and an error message should appear close to the submit button 💯